### DISCUSSION TITLE
دیوار یادبود

### DISCUSSION THREAD

**Discussion by AlliNabavieh** [WeAreMahsaAmini/Core#118] [Ideas]
با سلام به همه.
من فکر میکنم اگر بشه یک فضایی رو تشکیل داد از اسامی و عکس و مشخصات افرادی که توی این روزها جونشون رو از دست دادند یا به هر دلیلی ناپدید شدند٬ ( البته از طریق کانالهای مطمعن و درست ) اینجوری میشه به تمام دنیا نشون داد که ج.ا چطور و چجوری جوون ها رو کشته و نشون بدیم که همه بدونن الان مثلا این افراد ناپدید شدند و اونهایی که پیدا میشن رو مشخص بکنی.
اطلاعاتی که میشه برای نمایش گذاشت :
- نام و نام خانوادگی.
- جنسیت.
- سن.
- عکس یا فیلم.
- محل کشته شدن.
- اگر مفقود شده محل مفقودی.
- یک بیوگرافی کوتاه از فرد.
- اگر مشخص که در کجا زندانی شده محل اون و مشخصاتی که در دسترس هست.
- اگر نیاز به وکیل یا مشاور دارند گزینه داشته باشه که افرادی که میخوان کمک بکنند بتونن کمک بکنن. ( در بعضی مواقع ممکنه این موضوع گسترش پیدا بکنه و حتی سرپرست های خانوار براشون مشکل پیش بیاد و اگر نیاز به کمک های مردی بود قابلیت دونیت داشته باشه )
- ...

و در نهایت میتونه این اسامی یک دیوار بزرگ فلزی و یادبود باشه از کشته شده های این روزها و حتی بصورت فیزیکی اون رو هنرمندها بسازن و تو محل های مختلف نصب بکنن برای یادبود.

این یک چیز خیلی کلی بود از چیزی که به ذهن من رسید.
الان فرمتی که به ذهنم رسید مشابه همون دیوار همایتی از طرف افراد مشهور هست ولی میتونه از اون هم متفاوت باشه.

**Comment by ramyareye** [comment]
سلام
من یه نمونه اولیه برای این زدم تا امشب میذارم بالا 
تنها موردی که وجود داره دیتابیس هست
میشه همرو داخل یه فایل json گذاشت ولی بنظرم خیلی جالب نیست، اگر پیشنهادی دارید بدید روش کار کنم ممنون

**Reply by AlliNabavieh** [reply]
سلام ارادت.
راستش من برنامه نویسی خیلی نمیدونم و راهکاری برای اون بخشش ندارم متاسفانه.
ولی در بخش طراحی و UI/UX اگر بتونم خوشحال میشم کمکی بکنم و حتی میشه راهکارهایی برای پیدا کردن اسامی این عزیزان پیدا کرد.

**Reply by ramyareye** [reply]
من نسخه اولیه رو با یک دیزاین ساده می ذارم بالا شما می تونید روی اون کار کنید و ما پیاده سازی کنیم
در مورد بحث فنی هم من سعی می کنم راهکار مناسب پیدا کنم

**Reply by AlliNabavieh** [reply]
خیلی هم عالی.
حالا من دارم با یکسری از دوستانم هم صحبت میکنم اگر بشه اضافه بکنمشون به اینجا اگر کمکی یا چیزی لازم بود ازشون استفاده بکنیم.

**Reply by AlliNabavieh** [reply]
البته یک موضوع مهم دیگه اینکه آیا منتشر کردن این اطلاعات برای خانواده هاشون مشکل ساز نمیشه ؟
منظورم اونایی که مردند نیست٬ منظورم اون افرادی هست که زندانی شدن و یا خبری ازشون نیست.

چون این موارد رو هم باید در نظر بگیریم ( چون خیلی از این افراد تهدید میشن که هیچ صحبتی نکنن و هیج عکس العملی نشون ندن ) 
این رو چطور میشه بررسی کرد؟ آیا با خانواده ها باید مطرح کرد و ازشون بصورت الکترونیک یک تاییدیه گرفت بابت ثبت عکس و فیلم و مشخصاتشون یا بر اساس سوشال مدیا اول بارگذاری بشه و بعد اگر ناراضی بودند پاک بشه ؟؟

**Reply by AlliNabavieh** [reply]
@ramyareye شما هم رو این موضوع یکم فکر بکن ببین چه مسیری رو میشه در نظر گرفت برای چند باره کاری نشدن موضوع.

البته موضوع رضایت الکترونیک و این داستانا تو این شرایط یکم سخت میکنه کار رو ( چون یکسری فرمول اهراز هویت و این داستانا داره که خب الان منطقی و شدنی نیست.)

**Reply by kaveh6** [reply]
من به شخصه فکر میکنم اگر کسی رو میگیرن ، شانسش خیلی بیشتره زمانی که اسم و مشخصاتش معلوم باشه و منتشر بشه تا زمانی که ازش هیچ اسمی نباشه .  
و البته علاوه بر اون خیلی هم میتونه تاثیر گذار باشه این حرکت روی بقیه ، علی الخصوص دوستان و آشناها و فامیلش .

**Reply by ksaffarian** [reply]
ایده خیلی خوب و مثبتی هست.
فکر میکنم خیلی از این صفحات رو بشه بصورت یک وب سایت ساده که محتوا داخل گیت هاب باشه و از طریق PR اپدیت بشه مدیریت کرد.
در نهایت با راهکارهایی مثل https://docusaurus.io
رندر و منتشر کرد.
)البته من قبلا اینکار رو نکردم)
به نظرم اساسا نباید دنبال راهکارهای پیچیده باشیم وگرنه نگهداری اش ممکنه سخت باشه.

**Reply by AlliNabavieh** [reply]
@ksaffarian کلا برای استارتش باید خیلی ساده دیده بشه.
کلا در طراحی اپلیکیشن یا استارت آپ هم ما این رو داریم که اول با MVP و نمونه اولیه بیایم بالا و بر اساس بازخورد و به مرور بتونیم تکمیل تر و بهتر پرورشش بدیم و بتونیم بیشتر به معرض دید عموم بزاریم.

این وب سایت میتونه به خیلی ها کمک بکنه و صدای تمام این جان باختگان و زندانی های سیاسی باشه ( هم برای قبلی ها و هم برای این دوره و امیدوارم دیگه بعدی هایی در کار نباشه ) 
کلا هم خیلیی ساده باید باشه در ابتدا.
اسم و عکس و شهر و مشخصات فردی و یک نوشته کوتاهی از فرد که بفهمونیم این فرد واقعی است و یک هویت واقعی داره و فیک نیست.
برای زندانی ها هم میشه خیلی اطلاعات مفید تری رو منتشر کرد.
در ساده ترین حالت ممکن البته در ابتدای کار.

**Reply by AlliNabavieh** [reply]
> سلام من یه نمونه اولیه برای این زدم تا امشب میذارم بالا تنها موردی که وجود داره دیتابیس هست میشه همرو داخل یه فایل json گذاشت ولی بنظرم خیلی جالب نیست، اگر پیشنهادی دارید بدید روش کار کنم ممنون

سلام ارادت.
به نتیجه ای رسیدید دیشب تونستید یک نمونه اولیه رو بالا بیارید؟

**Comment by daneial** [comment]
کمکی خواستید  در مورد هر پروژه‌ای منم هستم
طراح محصولم هست تخصصم.

**Reply by AlliNabavieh** [reply]
خیلی هم عالی .
مرسی از پیشنهاد کمکتون.
راستش فعلا نیاز به یک طرح اولیه از انی پلتفرم داریم و یکسری موارد قانونی اگر کسی اطلاع داشته باشه که مشکلی برای کسی ایجاد نشه و اطلاعات اشتباه منتشر نکنیم که بعدا دردسر ساز بشه.

**Reply by daneial** [reply]
من اوکیم اگه دیدید کمکی از دست من برمیاد در خدمتم
در جریانم بزارید حتما

**Comment by BaharehUlogic** [comment]
great idea. I'm java developer, and I can coding With Html ,thymeleaf , how i can help?

**Reply by AlliNabavieh** [reply]
Thanks for your suggestion.

Actually we need a MVP for this idea "Maybe just an idea in Figma" could be enough in this stage.
I'm not a developer so I just try to sketch a template for this platform this weekend.
if you have an idea or example for this idea please let us know.

**Comment by ramyareye** [comment]
guys I made the very first working version in here : https://memorial-thewicked.vercel.app/

Database : back4app (we may stay here or move let's see) (free for a good range of data)
Web App: Nextjs hosted on Vercel (free) 
Data Proxy: Cloudflare Workers (free for our usecase)

repos: 
https://github.com/ramyareye/fm (nextjs)
https://github.com/ramyareye/fm-worker (cfw)

please share your thoughts and let's discuss and push, thanks
@syavash

**Reply by whiteh7t** [reply]
Although I think this draft is too simple and needs many UI enhancements as you know, it's definitely nice to start. I suggest placing all related data in HTML tags, not on pictures because SEO matters.

**Comment by ghost** [comment]
سلام.
من میتونم کمک کنم برای این کار، طراحی وبسایت و اینا رو بلد نیستم ولی برای جمع آوری اطلاعات میتونم کمک کنم.

میخواستم پیشنهاد کنم صفحات ویکیپدیا هم درست کنیم، اینجوری میشه تعداد بیشتری آدم رو درگیر کرد

**Reply by AlliNabavieh** [reply]
سلام مرسی از پیشنهادتون. 
فکر‌خوبیه ولی من خیلی ویکیپدیا روبلدش نیستم ولی شنیدم که همه میتونن اطلاعات توش بگذارند ( یعنی یکجورایی open source ) هست و خب این به نظر شما نمیتونه دستکاری بشه ؟ 
من خیلی وارد نیستم تو‌این زمینه ولی اگر اینطور باشه شاید خیلی منبع خوبی برای ارائه نباشه. 

الان مثلا افرادی که توی پرواز اوکراینی فوت کردند قطعا یک لیست دقیق و درستی ازشون وجود داره که میشه از اونجا استناد کرد. 
همینطور برای داستان‌های قبلی اگر بشه لیست تهیه کرد و همه رو مستند کرد خیلی عالی میشه.

**Comment by Arefeh902** [comment]
سلام
من توی فکر دیوار یادگاری بودم ولی چیزی که توی ذهنم بود یکم متفاوت بود.

- یادگاری برای کسانی که جونشون رو از دست دادن باشه (مشکل اینکه خانواده بازداشتی ها ممکنه نخوان رسانه ای بشه نمونش دوست خودم یا ممکنه پیچیدگی بیاره یا هر چیزی)
- روایت کاملشون باشه - مخصوصا تکه ی دستگیری و اخبار و همه مطالب بعدشون - همه و همه یه جا جمع باشه و حتی توییت ها و لینک خبر ها و مصاحبه ها و همه چیز
- یه مشکل ممکنه پیش بیاد اینکه برای بعضی از افراد مطلب و توییت و این مسائل زیاد زده شده 
برای اونا خوبه یه سیستم باشه که آدم ها رای بدن و اینا رنک شن بر اون اساس 
و کلا هم خوبه که مطالب تکراری تا حدی فیلتر شن (برای به گوش مخاطب کم حوصله رسوندن)
که می تونه یه رنکینگ ساده باشه که باید بیشتر دربارش فکر بشه
- برای مطالب اشتباه هم بحث حقوقیش رو نمی دونم ولی چند تا ادمین و گزینه ریپورت می تونه باعث تشخیص اشتباهات احتمالی بشه

به جهت فنی هم من هستم (تکنولوژی و ایناش رو می تونیم صحبت کنیم)

**Reply by AlliNabavieh** [reply]
سلام ارادت.

خیلی خوبه و ترکیب این ۲ تا ایده ( که ماهیت یکسانی دارند ) میتونه تبدیل به یک منبع بسیار قدرتمند بشه برای اینکه بتونه اطلاع رسانی خیلی مفیدی باشه برای عزیزانی که زندانی هستند یا خبری ازشون نیست (و یادبود و اطلاع رسانی باشه برای افرادی که جونشون رو از دست دادند ( بر اساس روایت و شواهد و به قول شما توییت و خبر و ... ) 
و در این بین خیلی ها بودند که جونشون رو از دست دادند و خبر آنچنانی درباره اشان گذاشته نشده ( و این کمی کم لطفی هست به خانواده هاشون و ممکنه باعث دلسردی اونها بشه.

به نظرم باید یک بستری ایجاد بشه بتونیم این ایده های پراکنده رو دسته بندی بکنیم و شروع کار رو انجام بدیم.

ولی خب اصل مطلب این هست که چطور میشه اینکار رو کرد؟
من خودم در زمینه فنی خیلی اطلاعات کاملی ندارم و تیم اجرایی رو باید جمع آوری کرد.
ولی در زمینه ایده پردازی و طراحی رابط کاربری میتونم کمک بکنم برای این موضوع و برای جمع آوری اطلاعات میتونیم از کانال های مطمعن خبری یکسری اطلاعات جمع آوری بکنیم.
و تنها موضوعی که الان ممکنه مشکل ساز بشه برای تیم اجرایی اسم و مشخصات افرادی هست که قرار است روی این پروژه کار بکنند ( از خریداران هاست و دامنه گرفته تا مجری ها و ... ) اینکه چه راهکاری رو میشه چید که این افراد با خیال راحت کار بکنند نکته مهمی هست.

**Comment by arashi-dev** [comment]
من میتونم تو زمینه فرانت کمک کنم. typescript, NextJS
کمی دانش nodejs هم دارم اگر نیاز به همراه هست

**Comment by ramyareye** [comment]
دوستان ممنون از همراهیتون
من با سیاوش صحبت کردم و یک پروژه تعریف کردیم و من باید کد اولیه رو به همراه یکسری تغییرات پوش بدم
متاسفانه مقداری درگیر شدم، طی ۱-۲ روز آینده میفرستم بالا حتما و اینجا اعلام می کنم 
یک پروژه براش میسازیم داخل دیسکورد و جلسات و هماهنگی‌هارو انجام میدیم
